wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/CellDescription/MLSMigrationCellDescription.swift

```diff
@@ -92,59 +101,83 @@ final class MLSMigrationCellDescription: ConversationMessageCellDescription {
         typealias Localizable = L10n.Localizable.Content.System.MlsMigration
 
         let text = NSMutableAttributedString.markdown(
-            from: Localizable.Started.description,
+            from: SystemMessageMLSMigrationLocale.Started.description,
             style: .systemMessage
         )
         let link = NSAttributedString(
-            string: Localizable.learnMore,
+            string: SystemMessageMLSMigrationLocale.learnMore,
             attributes: linkAttributes
         )
         return [text, link].joined(separator: NSAttributedString(" "))
     }
 
     private static func makeOngoingCallAttributedString() -> NSAttributedString? {
         NSAttributedString.markdown(
-            from: L10n.Localizable.Content.System.MlsMigration.ongoingCall,
+            from: SystemMessageMLSMigrationLocale.ongoingCall,
             style: .systemMessage
         )
     }
 
     private static func makeUpdateVersionAttributedString() -> NSAttributedString? {
         NSAttributedString.markdown(
-            from: L10n.Localizable.Content.System.MlsMigration.Started.updateLatestVersion,
+            from: SystemMessageMLSMigrationLocale.Started.updateLatestVersion,
             style: .systemMessage
         )
     }
 
     private static func makeJoinAfterwardsAttributedString() -> NSAttributedString? {
-        typealias Localizable = L10n.Localizable.Content.System.MlsMigration
 
         let text = NSMutableAttributedString.markdown(
-            from: Localizable.joinAfterwards,
+            from: SystemMessageMLSMigrationLocale.joinAfterwards,
             style: .systemMessage
         )
         let link = NSAttributedString(
-            string: Localizable.learnMore,
+            string: SystemMessageMLSMigrationLocale.learnMore,
             attributes: linkAttributes
         )
         return [text, link].joined(separator: NSAttributedString(" "))
     }
 
     private static func makePotentialGapAttributedString() -> NSAttributedString? {
-        typealias Localizable = L10n.Localizable.Content.System.MlsMigration
 
         let text = NSMutableAttributedString.markdown(
-            from: Localizable.potentialGap,
+            from: SystemMessageMLSMigrationLocale.potentialGap,
             style: .systemMessage
         )
 
         let link = NSAttributedString(
-            string: Localizable.learnMore,
+            string: SystemMessageMLSMigrationLocale.learnMore,
             attributes: linkAttributes
         )
 
         return [text, link].joined(separator: NSAttributedString(" "))
 
     }
 
+    private static func makeMLSNotSupportedForSelfUser(username: String) -> NSAttributedString? {
+
+        let text = NSMutableAttributedString.markdown(
+            from: SystemMessageMLSMigrationLocale.mlsNotSupportedByYou(username),
+            style: .systemMessage
+        )
+
+        let link = NSMutableAttributedString(
+            string: SystemMessageMLSMigrationLocale.Download.Mls.wire,
```

Comment:
from the figma it seems only one part of the link is bold: "Download the lastest MLS Wire version". here it seems you set everything as link. Was it cleared with design?